server/src/entity/recipe.ts

```diff
@@ -13,32 +13,134 @@ export class Recipe extends BaseEntity<Recipe> {
     @MinLength(3)
     public title: string;
 
+    @Column()
+    public vegetarian: boolean;
+
+    @Column()
+    public vegan: boolean;
+
+    @Column()
+    public glutenFree: boolean;
+
+    @Column()
+    public dairyFree: boolean;
+
+    @Column()
+    public veryHealthy: boolean;
+
+    @Column()
+    public cheap: boolean;
+
+    @Column()
+    public veryPopular: boolean;
+
+    @Column()
+    public sustainable: boolean;
+
     @Column({ nullable: true })
-    public description: string;
+    public weightWatcherSmartPoints: number;
+
+    @Column()
+    public gaps: string;
+
+    @Column()
+    public lowFodmap: boolean;
+
+    @Column()
+    public ketogenic: boolean;
+
+    @Column()
+    public whole30: boolean;
+
+    @Column()
+    public preparationMinutes: number;
+
+    @Column()
+    public cookingMinutes: number;
+
+    @Column()
+    public sourceUrl: string;
+
+    @Column()
+    public spoonacularSourceUrl: string;
+
+    @Column()
+    public aggregateLikes: number;
+
+    @Column()
+    public spoonacularScore: number;
+
+    @Column()
+    public healthScore: number;
+
+    @Column()
+    public creditsText: string;
+
+    @Column()
+    public sourceName: string;
+
+    @Column()
+    public readyInMinutes: number;
+
+    @Column()
+    public pricePerServing: number;
+
+    @Column()
+    public servings: number;
```

Comment:
Is it correct that I'm missing properties? Perhaps it is better to use the same names and order as Spoonacular.